assets/js/modules/adsense/datastore/tags.js

```diff
@@ -73,20 +57,101 @@ const fetchGetTagPermissionStore = createFetchStore( {
 	},
 } );
 
+// Actions
+const GET_EXISTING_TAG = 'FETCH_EXISTING_TAG';
+const RECEIVE_EXISTING_TAG = 'RECEIVE_EXISTING_TAG';
+const WAIT_FOR_EXISTING_TAG = 'WAIT_FOR_EXISTING_TAG';
+
 const BASE_INITIAL_STATE = {
 	existingTag: undefined,
 	tagPermissions: {},
 };
 
+const baseActions = {
+	getExistingTag() {
+		return {
+			payload: {},
+			type: GET_EXISTING_TAG,
+		};
+	},
```

Comment:
Let's use `fetchGetExistingTag` and `FETCH_GET_EXISTING_TAG`, sticking to our conventions to always use "fetch" and "receive" for these kinds of internal actions.

Reply:
I thought about that, but the `fetch` is really the responsibility of the `core/site` store, the module isn't really concerned with where it comes from, it only needs HTML to parse it from. It seems inaccurate to call a `FETCH_` action that doesn't itself perform a fetch.

Reply:
While this action itself does not perform a `fetch`, technically a `fetch` is performed though. Per our naming conventions I don't think it makes sense to have anything but a public selector start with `get...`. Our "fetch" and "receive" prefixes indicate that these are internal actions that fetch and receive data from some URL, which applies here too - if we had not abstracted this out into a `getHTMLForURL` in `core/site`, the `fetch` request would happen here.